index.html

```diff
@@ -281,9 +281,10 @@ <h2>Acknowledgements</h2>
     <p>
 Portions of the work on this specification have been funded by the United States
 Department of Homeland Security's Silicon Valley Innovation Program under
-contracts 70RSAT20T00000010, and 70RSAT20T00000029. The content of this
-specification does not necessarily reflect the position or the policy of the
-U.S. Government and no official endorsement should be inferred.
+contracts 70RSAT20T00000010, 70RSAT20T00000029, 70RSAT20T00000003, and
+70RSAT20T00000033. The content of this specification does not necessarily
```

Comment:
Making it easier to check for duplicates and/or presence of contract references when considering adding more...
```suggestion
contracts
70RSAT20T00000003,
70RSAT20T00000010,
70RSAT20T00000029,
and
70RSAT20T00000033.
The content of this specification does not necessarily
```

Reply:
Hi @TallTed , i modified your comment to add our contracts hope you don't mind. 
Excellent suggestion!

Reply:
@mkhraisha -- Fine by me, though I'm now tweaking it again to maintain the alphanumeric ordering, which is what makes it easier to prevent duplicates.